docs/features/theme.stories.mdx

```diff
@@ -18,7 +18,7 @@ import { Heading, SubHeading, Text } from '../../packages/circuit-ui';
 
 <Intro>
 
-The theme used throughout Circuit UI comes from SumUp's design token library, [`@sumup/design-tokens`](Packages/design-tokens), which is a required peer dependency of `@sumup/circuit-ui`. Refer to its [documentation](Packages/design-tokens) for usage instructions.
+The theme used throughout Circuit UI comes from SumUp's design token library, [`@sumup/design-tokens`](https://github.com/sumup-oss/circuit-ui/tree/main/packages/design-tokens), which is a required peer dependency of `@sumup/circuit-ui`. Refer to its documentation for usage instructions.
```

Comment:
Same as above.

Reply:
done

Reply:
I still see the change 🤔

Reply:
forgot that was my remote for push was in my fork